CMakeLists.txt

```diff
@@ -1527,7 +1527,7 @@ if (ENABLE_UNITTESTS AND ENABLE_CXX11)
 		#set_tests_properties(test-srt PROPERTIES RUN_SERIAL TRUE)
 	else()
 		set_tests_properties(${tests_srt} PROPERTIES RUN_SERIAL TRUE)
-		gtest_discover_tests(test-srt)
+		#gtest_discover_tests(test-srt)
```

Comment:
Is CodeCov using the only possible default configuration, or it can be done with enabled on demand?

Here I can withdraw this change, but enabling these tests on demand (and not by default) is required.